core/sail/nativerdf/src/main/java/org/eclipse/rdf4j/sail/nativerdf/model/CorruptValue.java

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Eclipse RDF4J contributors, Aduna, and others.
```

Comment:
This is an old copyright header. See https://github.com/eclipse-rdf4j/rdf4j/blob/main/CONTRIBUTING.md#code-formatting